### DISCUSSION TITLE
Didn‘t get any records

### DISCUSSION THREAD

**Discussion by p1337** [jekhokie/raspberry-noaa-v2#275] [Q&A]
Hello there, maybe someone could help me.

System Raspi 4 
Raspi OS 
installad Raspberry pi Noaa V2 with auto-script.

tried Noxon DAB+ Stick V1 
and Noelec SDR Stick

The predictions and Schedules are made.
did not get any NOAA or METEOR images or audio files.

EDIT 14.03.2021 - 20:00 : maybe look at the past 18:30 log -> i have played arround a bit before.
 

this is my output.log:
`14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Starting rtl_fm record
ERROR : Satellite  is not valid - please use one of ['NOAA 15', 'NOAA 18', 'NOAA 19'].
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/audio_processors/noaa_record.sh ERROR : Satellite  is not valid - please use one of ['NOAA 15', 'NOAA 18', 'NOAA 19'].
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Producing spectrogram
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Bulding pass map
wxmap: error: : Is a directory
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-ZA.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-ZA.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-ZA.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-ZA.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-MCIR.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-MCIR.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-MCIR.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-MCIR.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-MCIR-precip.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-MCIR-precip.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-MCIR-precip.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-MCIR-precip.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-therm.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-therm.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-therm.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-therm.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Total processing time: 00:00.06
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Clearing and re-creating TLE file with latest...
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Clearing existing scheduled 'at' capture jobs...
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling new capture jobs...
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling NOAA 15 captures...
INFO : Scheduling capture for: NOAA-15 20210314-181540 43
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule_captures.sh INFO : Scheduling capture for: NOAA-15 20210314-181540 43
warning: commands will be executed using /bin/sh
job 9 at Sun Mar 14 18:15:00 2021
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling NOAA 18 captures...
INFO : Scheduling capture for: NOAA-18 20210314-204217 51
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule_captures.sh INFO : Scheduling capture for: NOAA-18 20210314-204217 51
warning: commands will be executed using /bin/sh
job 10 at Sun Mar 14 20:42:00 2021
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling NOAA 19 captures...
INFO : Scheduling capture for: NOAA-19 20210314-175407 68
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule_captures.sh INFO : Scheduling capture for: NOAA-19 20210314-175407 68
at: refusing to create job destined in the past
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling Meteor-M 2 captures...
INFO : Scheduling capture for: METEOR-M-2 20210314-194757 48
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule_captures.sh INFO : Scheduling capture for: METEOR-M-2 20210314-194757 48
warning: commands will be executed using /bin/sh
job 11 at Sun Mar 14 19:47:00 2021
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Done scheduling jobs!
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Starting rtl_fm record
ERROR : Satellite  is not valid - please use one of ['NOAA 15', 'NOAA 18', 'NOAA 19'].
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/audio_processors/noaa_record.sh ERROR : Satellite  is not valid - please use one of ['NOAA 15', 'NOAA 18', 'NOAA 19'].
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Producing spectrogram
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Bulding pass map
wxmap: error: : Is a directory
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-ZA.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-ZA.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-ZA.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-ZA.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-MCIR.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-MCIR.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-MCIR.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-MCIR.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-MCIR-precip.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-MCIR-precip.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-MCIR-precip.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-MCIR-precip.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Decoding image
wxtoimg: error: could not open overlay map file '/home/pi/raspberry-noaa-v2/tmp/map/-map.png'
wxtoimg: error: could not open input file /srv/audio/noaa/.wav: No such file or directory
(standard_in) 1: syntax error
/home/pi/raspberry-noaa-v2/scripts/image_processors/noaa_normalize_annotate.sh: line 33: [: -eq: unary operator expected
Loading page (1/2)
Rendering (2/2)                                                    
Done                                                               
convert: unable to open image `/srv/images/-therm.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: ImageSequenceRequired `-composite' @ error/mogrify.c/MogrifyImageList/7995.
convert: no images defined `/srv/images/-therm.jpg' @ error/convert.c/ConvertImageCommand/3258.
convert: unable to open image `/srv/images/-therm.jpg': No such file or directory @ error/blob.c/OpenBlob/2874.
convert: no images defined `/srv/images/thumb/-therm.jpg' @ error/convert.c/ConvertImageCommand/3258.
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Total processing time: 00:00.04
14-03-2021 18:03 /home/pi/raspberry-noaa-v2/scripts/audio_processors/noaa_record.sh ERROR : Satellite  is not valid - please use one of ['NOAA 15', 'NOAA 18', 'NOAA 19'].
14-03-2021 18:15 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh ERROR : There is an existing rtl_fm instance running, I quit
14-03-2021 18:06 ./scripts/testing/test_reception.sh ERROR : Usage: ./scripts/testing/test_reception.sh <frequency>. Example: ./scripts/testing/test_reception.sh 90.3
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Clearing and re-creating TLE file with latest...
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Clearing existing scheduled 'at' capture jobs...
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling new capture jobs...
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling NOAA 15 captures...
date: invalid date '@'
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling NOAA 18 captures...
date: invalid date '@'
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling NOAA 19 captures...
date: invalid date '@'
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Scheduling Meteor-M 2 captures...
date: invalid date '@'
14-03-2021 18:31 /home/pi/raspberry-noaa-v2/scripts/schedule.sh INFO : Done scheduling jobs!
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO : Applying any schema updates...
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO :   - 00_seed_schema.sql already applied
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO :   - 01_add_spectrogram_bool.sql already applied
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO :   - 02_add_noaa_pristine_bool.sql already applied
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO :   - 03_add_pass_azimuth_direction.sql already applied
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO :   - 04_add_capture_gain.sql already applied
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO :   - 05_add_pass_azimuth_at_max.sql already applied
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO :   - 06_add_polar_az_el_bool.sql already applied
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/db_migrations/update_database.sh INFO : Schema updates complete!
14-03-2021 18:44 ./scripts/schedule.sh INFO : Clearing and re-creating TLE file with latest...
14-03-2021 18:44 ./scripts/schedule.sh INFO : Clearing existing scheduled 'at' capture jobs...
14-03-2021 18:44 ./scripts/schedule.sh INFO : Scheduling new capture jobs...
14-03-2021 18:44 ./scripts/schedule.sh INFO : Scheduling NOAA 15 captures...
14-03-2021 18:44 ./scripts/schedule.sh INFO : Scheduling NOAA 18 captures...
INFO : Scheduling capture for: NOAA-18 20210314-194216 51
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/scripts/schedule_captures.sh INFO : Scheduling capture for: NOAA-18 20210314-194216 51
warning: commands will be executed using /bin/sh
job 12 at Sun Mar 14 19:42:00 2021
14-03-2021 18:44 ./scripts/schedule.sh INFO : Scheduling NOAA 19 captures...
14-03-2021 18:44 ./scripts/schedule.sh INFO : Scheduling Meteor-M 2 captures...
INFO : Scheduling capture for: METEOR-M-2 20210314-184756 48
14-03-2021 18:44 /home/pi/raspberry-noaa-v2/scripts/schedule_captures.sh INFO : Scheduling capture for: METEOR-M-2 20210314-184756 48
warning: commands will be executed using /bin/sh
job 13 at Sun Mar 14 18:47:00 2021
14-03-2021 18:44 ./scripts/schedule.sh INFO : Done scheduling jobs!
14-03-2021 18:47 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : I'll flip this image pass because FLIP_METEOR_IMG is set to true
14-03-2021 18:47 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Starting rtl_fm record
INFO : Recording at 137.1000 MHz...
14-03-2021 18:47 /home/pi/raspberry-noaa-v2/scripts/audio_processors/meteor_record_rtl_fm.sh INFO : Recording at 137.1000 MHz...
Found 1 device(s):
  0:  NOXON, DAB Stick, SN: 0

Using device 0: Terratec NOXON DAB/DAB+ USB dongle (rev 1)
Found Fitipower FC0013 tuner
Tuner gain set to 7.10 dB.
activated bias-T on GPIO PIN 0
Tuned to 137388000 Hz.
Oversampling input by: 4x.
Oversampling output by: 1x.
Buffer size: 7.11ms
Sampling at 1152000 S/s.
Output at 288000 Hz.
Allocating 15 zero-copy buffers
pi@raspberrypi:~ $ 
`

**Comment by jekhokie** [comment]
@p1337 could you please run the `./support.sh` script and post the results here?

Also, those errors early in the logs about the satellite scheduling not being valid for the satellites chosen seems...odd. Have you modified any local files?

I should add - it's easier if you join us on Discord chat - we can talk in real time/help troubleshoot: https://discord.gg/5UfPraGh

**Reply by p1337** [reply]
Thank you for your fast reply.

i changed nothing but the copied config file. 

```
pi@raspberrypi:~/raspberry-noaa-v2 $ ./support.sh 
./support.sh: line 15: warning: command substitution: ignored null byte in input
=============================================
Details about environment
=============================================
Current date/time:  1615751558
Repo git hash:      ac134d390592f209a08f782a449348bdd938e78a
Raspberry Pi Model: Raspberry Pi 4 Model B Rev 1.2
Architecture:       Linux raspberrypi 5.10.17-v7l+ #1403 SMP Mon Feb 22 11:33:35 GMT 2021 armv7l GNU/Linux
Num CPUs:           4
---------------------------------------------
USB Device Map:
  Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
  Bus 001 Device 003: ID 0ccd:00b3 TerraTec Electronic GmbH NOXON DAB/DAB+ Stick
  Bus 001 Device 002: ID 2109:3431 VIA Labs, Inc. Hub
  Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
---------------------------------------------
Disk Info:
  NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
  mmcblk0     179:0    0 59.5G  0 disk 
  |-mmcblk0p1 179:1    0  256M  0 part /boot
  `-mmcblk0p2 179:2    0 59.2G  0 part /
---------------------------------------------
Disk Usage Info:
  Filesystem     1K-blocks    Used Available Use% Mounted on
  /dev/root       61078228 5345340  53199084  10% /
  devtmpfs         1827468       0   1827468   0% /dev
  tmpfs            1959564       0   1959564   0% /dev/shm
  tmpfs            1959564   16896   1942668   1% /run
  tmpfs               5120       4      5116   1% /run/lock
  tmpfs            1959564       0   1959564   0% /sys/fs/cgroup
  tmpfs            1024000       0   1024000   0% /var/ramfs
  /dev/mmcblk0p1    258095   48784    209312  19% /boot
  tmpfs             391912       0    391912   0% /run/user/1000
---------------------------------------------
Memory Info:
  MemTotal:        3919132 kB
  MemFree:         3668232 kB
  MemAvailable:    3707716 kB
---------------------------------------------
Git source files changed:
  (None)
---------------------------------------------
SDR Information:
Found 1 device(s):
  0:  Terratec NOXON DAB/DAB+ USB dongle (rev 1)

Using device 0: Terratec NOXON DAB/DAB+ USB dongle (rev 1)
Found Fitipower FC0013 tuner

Current configuration:
__________________________________________
Vendor ID:              0x0ccd
Product ID:             0x00b3
Manufacturer:           NOXON
Product:                DAB Stick
Serial number:          0
Serial number enabled:  yes
IR endpoint enabled:    no
Remote wakeup enabled:  yes
__________________________________________
```

**Comment by Cirromulus** [comment]
```
14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/receive_noaa.sh INFO : Starting rtl_fm record ERROR : Satellite is not valid - please use one of ['NOAA 15', 'NOAA 18', 'NOAA 19']. 14-03-2021 18:02 /home/pi/raspberry-noaa-v2/scripts/audio_processors/noaa_record.sh ERROR : Satellite is not valid - please use one of ['NOAA 15', 'NOAA 18', 'NOAA 19'].
```
This seems to be the problem.
What does your latest at-job say?
`atq`, then pick a number and type `at -c <number>`.
It seems that a new commit broke the interface between schedule and receive scripts

**Reply by p1337** [reply]
atq  has no output i think there are actually no passes

**Reply by Cirromulus** [reply]
Can you try now?
Yesterday in your timezone there were no passes left of the day, i recon
